guides/installing_esphome.rst

```diff
@@ -49,19 +49,37 @@ with the following:
 Mac
 ---
 
-There are no tested installation instructions for Mac. ESPHome does support
-Mac & will run with no problem.
+ESPHome supports macOS and will run without issues.
 
-Contributions are welcome!
+An easy way for installation is via `Homebrew <https://brew.sh/>`_:
```

Comment:
I'd suggest listing here the installation techniques - brew, pip and cloning the repo. Then document each method subsequently with a heading for each one.